docroot/modules/custom/sitenow_migrate/sitenow_migrate.install

```diff
@@ -25,3 +25,4 @@ function sitenow_migrate_uninstall() {
   }
   return TRUE;
 }
+*/
```

Comment:
Should we just remove this. Git will have it if we need to add it back in later.